src/App.js

```diff
@@ -0,0 +1,53 @@
+import './App.css';
+import InvoiceList from './InvoiceList';
+import Invoice from './Invoice';
+import { InvoicesContext } from './InvoicesContext';
+import data from './data.json';
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import iconLogo from "./assets/images/logo.svg";
+import iconMoon from "./assets/images/icon-moon.svg";
+import avatarIcon from "./assets/images/image-avatar.jpg";
+
+
+function App() {
+  const invoices = localStorage.getItem("invoiceList") ? JSON.parse(localStorage.getItem("invoiceList")) : data;
```

Comment:
```suggestion
  const invoices = localStorage.getItem('invoiceList') ? JSON.parse(localStorage.getItem('invoiceList)) : data;
```
Rekli smo da se u JS-u koriste `'` single quotes